tests/fixtures/ssl/client.certificate.pem

```diff
@@ -0,0 +1,89 @@
+Bag Attributes
+    friendlyName: client
+    localKeyID: 54 69 6D 65 20 31 36 36 34 38 30 39 34 32 35 31 35 39 
+Key Attributes: <No Attributes>
+-----BEGIN PRIVATE KEY-----
+MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQCzYMVhZAX5Swqs
+C/aahvDjiz3qHq83nHZU9e9MXv6njlqb06u5Olf7EMczH6/206xJ7SbJZJiyl9jF
+4KCvYRvzZ2KOfnsmMyT+6eThGHi/amWGsOaIdD2UbwpO3YxZkG6Z1R7Nst5aGRYN
+3H/q8eA+vOgtcCqUMApURfUCtTS81FHzhtbDY7Pnf3yUtSwhMEWTkuHBOcES6S2e
+PAkBu9augqsZ4GsJREwwN8o8M0dYyHtz/LRMRAjBAgFdmBOQHjBQkMMNJzR5xq5y
+KATX4szbeazsILAHLI1hrdJ6w/QK4kh67VSzWEPU94/iQjniMnY1yx09TL8R5b9H
+C5pnvozbAgMBAAECggEAIRit0nZeE80gvvXGHmTfdVxvpA7dTWl0++AUswaj/c5q
+MW4wtRBqw0wpNOmX0fbkRw+WpHoHIoyZ1ojHnRRRz+s3aAs2NaoiHVaSgGW/gbHj
+4aIqJusroYOLZ1nBV5Uqkv+bm2BxXjZat1wDzhmqwmigfiG/+W+TuFDgEAvXGy/R
+7QIZrOfGvEMfm0ZPU/12MNjO2APR9uXbgbZukTugm3uGj6VIlYvFkDuNaXfG2EL8
+85UnebaXWgWoFwibN+Ht8JH6DMXWrZgjWYTsK3Zjo7BdKN6CYQHClermf/dzUnYr
+HqFOHCB4sbndv8y35boxEDoZf5YYbkrNUFCVQosv4QKBgQDy0FVHO9AskHVMOJ5o
+YuOjMTIEhwpl+E3R9gCJX3ZSYdkGvN0dTory0Gw2ecqC1ip4wRKKqpEzh/WW66ZG
+cDB8kLnYxXDaR2DhwD1ekLaxY9ksCZV3HwIvt7npa8j0Z/nmuH1u4MgHE/7X21bE
+8vrBeCnckPYyRoU1/NLewTR8JwKBgQC9HokRM7VTQisMkoAOeIDA6Afz0g1FXgTo
+fvNB0lNds84Y/aly7jRUSwYkzTlA0132tDmdwethG6RLWfbVqt7ttH5dZGwuQgEs
+xr49Qvf1/jCeKNaxFXfx89m78iWGabbklFwmk1WeWRg7gZMjP0jWen4918oktaEo
+mXIdkFe2LQKBgHQ/aZSA/llXj9C+CpjPP0yR8PBZ9pB7IEa1D+y012D5l32yTeoK
+AFvfjO9Hv7yD7DGDvezFXF7faKSojSKL86qvX0p4vF7EbY2nC7qKbezfcMm0jOpK
+3jEaNNeb0smwPeRHz0WWOC2rVQbi69e+vG6DqQcAH95SBNSkBhC/TvFhAoGBAIGX
+Jv1unVXMBYu755wC3tdRKAHfcdCSuHJIa5Ay+4U97IAbcb2blKgeMLuPFShirgSb
+Qs/jTdNhCZfIseeACvp/sAd3BNo9yGhSV5HLDfa2k8IM4fC+cU6OdijfqO8OCZFg
+SDT2yLiU/JHqHFUOzG6LyW2/LHkzLJeRbizr9+6RAoGBANs7Sb07I8SVslTgGO9B
+pF50Mzqd3T2yELzhsbVD70Vzlkip5YAGj0ZbModbOdLRwwX5icSY8PWGq5xF7o/p
+5u0eaFWIqAjm0uB2ZfgI07mxJ/X1e68pcrcXpiPm7AZhUu8M05ZcTRaLXR10HJ60
+uyEnFW52+XSXj7N5LFyQmidT
+-----END PRIVATE KEY-----
+Bag Attributes
+    friendlyName: client
+    localKeyID: 54 69 6D 65 20 31 36 36 34 38 30 39 34 32 35 31 35 39 
+subject=C = US, ST = Ca, L = PaloAlto, O = CONFLUENT, OU = TEST, CN = client
```

Comment:
Can we create these with a self-signed certificate to avoid the inclusion of CONFLUENT?